### PR TITLE
Fix commit hook and improve it

### PR DESCRIPTION
**Pull request by Sartonon**
- Fix commit hook and improve it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,9 +69,11 @@
     "build": "react-scripts build",
     "lint": "eslint --ext js,ts,tsx src",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
-    "test:changed": "npm run test -- -o --watchAll=false",
+    "test:changed": "npm test -- -o --watchAll=false",
```
Comment (jorilindell) [inline package.json:72]: Maybe you could use yarn instead, but not big deal

File: package.json [hunk 2]
```diff
@@ -69,9 +69,11 @@
...
     "test:coverage": "react-scripts test --coverage --env=jest-environment-jsdom-sixteen --watchAll=false",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "test:nowatch": "react-scripts test --watchAll=false --env=jest-environment-jsdom-sixteen",
+    "test:staged": "npm run test:nowatch -- --findRelatedTests",
```
Comment (jorilindell) [inline package.json:76]: Maybe yarn?

File: package.json [hunk 3]
```diff
@@ -161,13 +163,21 @@
...
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run test:changed"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "src/**/*.{tsx,ts}": [
+      "npm run test:staged",
```
Comment (jorilindell) [inline package.json:178]: Maybe yarn?

**Review by jorilindell — APPROVED** [review]
Looks good

**Comment by Sartonon** [comment]
Changed npm run yarn

**Pull request merged**